src/pkg/oidc/helper.go

```diff
@@ -403,6 +403,10 @@ func groupsFromClaims(gp claimsProvider, k string) ([]string, bool) {
 		}
 		res = append(res, s)
 	}
+	// if remote.Groups is not a []string, but local.Groups is, this will create a fallback
```

Comment:
IMO, given remote/local are not declared in this func, this chunk should also be moved out...

Reply:
It makes sense, but IMO a comment in this location is necessary to explain this conditional.

What are your thoughts on changing the wording to something like

`in case all elements of the groupClaim are invalid, this creates a fallback to the next groupClaim`?

Reply:
> IMO, given remote/local are not declared in this func, this chunk should also be moved out...

It makes sense, but IMO a comment in this location is necessary to explain this conditional.

What are your thoughts on changing the wording to something like

`in case all elements of the groupClaim are invalid, this creates a fallback to the next groupClaim`?